seed/tests/test_property_views.py

```diff
@@ -1522,8 +1522,7 @@ def test_property_meters_endpoint_returns_a_list_of_meters_of_a_view(self):
         }
         gb_gas_meter = Meter.objects.create(**meter_details)
 
-        url = reverse('api:v3:properties-meters', kwargs={'pk': self.property_view_1.id})
-        url += f'?organization_id={self.org.pk}'
+        url = reverse('api:v3:property-meters-list', kwargs={'property_pk': self.property_view_1.id})
```

Comment:
existing API is now under `property-meters-list` and does not require the passing of the org_id.